nncf/initialization.py

```diff
@@ -97,12 +125,21 @@ def __init__(self, model, init_device: str):
 
     def _run_model_inference(self, data_loader, num_init_steps, device):
         bar_format = '{l_bar}{bar} |{n_fmt}/{total_fmt} [{elapsed}<{remaining}]'
+        # with tqdm(
```

Comment:
Commented code

Reply:
@vshampor it was a temoorary workaround. Will be fixed by #390